networks/rpc/handler.go

```diff
@@ -327,14 +327,14 @@ func (h *handler) handleCallMsg(ctx *callProc, msg *jsonrpcMessage) *jsonrpcMess
 	switch {
 	case msg.isNotification():
 		h.handleCall(ctx, msg)
-		logger.Debug("Served "+msg.Method, "duration", time.Since(start))
+		logger.Trace("Served "+msg.Method, "duration", time.Since(start))
```

Comment:
@JayChoi1736 Could you denote in PR or comment why the log level changed?

Reply:
added in PR. thanks :)